_api-reference/document-apis/pull-based-ingestion-management.md

```diff
@@ -77,10 +78,48 @@ The following table lists the available query parameters. All query parameters a
 | `cluster_manager_timeout` | Time units | The amount of time to wait for a connection to the cluster manager node. Default is `30s`. |
 | `timeout` | Time units | The amount of time to wait for a response from the cluster. Default is `30s`. |
 
+### Request body
+
+A list of reset settings can be provided for all or a subset of shards that need to be reset. This is optional.
+
+| Field | Data type | Required/Optional | Description |
+| :--- | :--- | :--- | :--- |
+| `shard` | Number | Required | The shard to reset. |
+| `mode` | String | Required | Reset mode. Allowed values are `offset` and `timestamp`. |
```

Comment:
Please add a short description of each value in parentheses. Should we link to [this doc](https://docs.opensearch.org/docs/latest/api-reference/document-apis/pull-based-ingestion/#stream-position) for more information?
```suggestion
| `reset_settings.mode` | String | Required | The reset mode. Valid values are `offset` and `timestamp`. |
```